docs/source/api/v2/topologies.rst

```diff
@@ -0,0 +1,593 @@
+..
```

Comment:
This file should be moved into the new `v3` directory, with examples referencing `/api/3.0`.

Reply:
Moved to `v3` directory in ba4d584054, examples reference `/api/3.0` in 61979e32a.